public/index.html

```diff
@@ -14,6 +14,18 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>QPP Submissions API Developer Documentation</title>
+    <!--START Google Analytics tracking code - for qpp.cms.gov-->
```

Comment:
we should update the domain referenced here to `cmsgov.github.io/qpp-submissions-docs`